crates/lib3h/src/dht/mod.rs

```diff
@@ -86,8 +86,8 @@ pub mod tests {
 
     #[allow(non_snake_case)]
     fn create_EntryData(
-        entry_address: &Address,
-        aspect_address: &Address,
+        entry_address: &EntryHash,
+        aspect_address: &AspectHash,
```

Comment:
do we want to rename the fields to entry_hash && aspect_hash? I'm not sure I have an opinion either way, just asking the question.

Reply:
no we dont. No point in specifying type in variable name. Its used as an address so name is fine